ANMN/LTSP/TSaggregator/aggregated_timeseries.py

```diff
@@ -188,13 +189,15 @@ def generate_netcdf_output_filename(fileURL, nc, VoI, file_product_type, file_ve
 
     file_timeformat = '%Y%m%d'
     nc_timeformat = '%Y%m%dT%H%M%SZ'
+    if '_' in VoI:
+        VoI = VoI.replace('_', '-')
     t_start = pd.to_datetime(nc.TIME.min().values).strftime(nc_timeformat)
     t_end = pd.to_datetime(nc.TIME.max().values).strftime(nc_timeformat)
```

Comment:
You can just use `file_timeformat` here. We don't really need the start and end times, just dates will do.

Reply:
changed.